emprofile

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
```

Comment:
You can create these while keeping the tool under tool if you like?

You can just add `tools/embuilder` to `tools/create_entry_points.py` ad then re-run it.

These scripts will then get created in the `tools` directory, which I think it probably fine. 

If you really want to make it so that folks don't have to type `./tools/empofile` but just `./emprofile` I think there are a few other places in the docs that will need updating.  

I think it might be simple to leave it there myself but I don't have strong feelings.

Reply:
I do want the users to be able to run just `emprofile` and not `$EMSCRIPTEN/tools/emprofile`, the earlier pattern to enable was too complex, and I had to always look at the doc page after a few months had passed since I used this tool. Now after this enabling the profiler should be much simpler.

Reply:
OK sounds reasonable.   Perhaps we need to modify create_entry_points.py to handle creating wrappers at the top level to tools in the `tools` directory.   I can take a look at this if you like?

Reply:
Actually it looks like this would make the bat/sh run_python scripts way less trivial.  Would you mind if we land this change without moving `emprofille` out of the `tools` directory. 

We can then followup with a plan to enable tools like `emdump` `emprofile` and `file_packager` to be moved to the top level at some point in the future?   It seems like we should have a generic solution here.

lgtm with `create_entry_points.py` usage and keeping launchers under tools.

Reply:
I have that other PR for ccache that already rewrites parts of the scripts creation, I'd prefer not to have to manage conflicts here. Hoping this could land as-is..?

Reply:
I took a look at solution for the create_entry_points.py problem: https://github.com/emscripten-core/emscripten/pull/13679
Hopefully it will make this change easier.